plume/src/main/kotlin/io/github/plume/oss/passes/type/GlobalTypePass.kt

```diff
@@ -44,8 +51,12 @@ class GlobalTypePass(private val driver: IDriver) : ITypePass {
             }
         return ts
     }
-
+    /*
+     * TYPE -(REF)-> TYPE_DECL
+     */
     private fun getGlobalTypeDecl(t: Type): NewNodeBuilder {
+        val shortName = if (t.toQuotedString().contains('.')) t.toQuotedString().substringAfterLast('.')
+        else t.toQuotedString()
         return nodeCache
```

Comment:
Over here we may need to add a filter to the `nodeCache`. So before the `.find` add:
```kotlin
.filterIsInstance<NewTypeDeclBuilder>()
```